.github/workflows/hardhat-core-ci.yml

```diff
@@ -58,6 +58,12 @@ jobs:
           command: check
           args: --workspace --all-features --all-targets
 
+      - name: Cargo check without all features
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --workspace --all-targets
```

Comment:
`--no-default-features`

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4037/commits/b94cf7c351d55e44618d3331c3301cedca20fe2f